tests/e2e-leg-5/metrics-auth-proxy-cert/05-deploy-operator.yaml

```diff
@@ -15,4 +15,4 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
   - command: sh -c "cd ../../.. && make undeploy-operator || true"
-  - command: sh -c "cd ../../.. && make deploy-operator DEPLOY_WITH=helm NAMESPACE=$NAMESPACE HELM_OVERRIDES='--set prometheus.tlsSecret=custom-cert,prometheus.expose=EnableWithAuthProxy'"
+  - command: sh -c "cd ../../.. && make deploy-operator DEPLOY_WITH=helm NAMESPACE=$NAMESPACE HELM_OVERRIDES='--set prometheus.tlsSecret=custom-cert,prometheus.expose=EnableWithAuthProxy,serviceAccountNameOverride=special-sa'"
```

Comment:
At some point in this test are we checking that `special-sa` exists?

Reply:
No, I don't check that. The operator will crash if the webhook ClusterRoleBinding is setup correctly. That was my thought process here. But I think checking specifically for the sa makes sense.